fre/cmor/cmor_mixer.py

```diff
@@ -141,20 +144,20 @@ def create_tmp_dir(outdir):
                 file output. tmp_dir will be slightly different depending on the output directory
                 targeted
     '''
-    print(f"(cmorize_target_var_files) outdir = {outdir}")
+    print(f"(create_tmp_dir) outdir = {outdir}")
     tmp_dir = None
     if any( [ outdir == "/local2",
               outdir.find("/work") != -1,
               outdir.find("/net" ) != -1 ] ):
-        print(f'(cmorize_target_var_files) using /local /work /net ( tmp_dir = {outdir}/ )')
-        tmp_dir = "{outdir}/"
+        print(f'(create_tmp_dir) using /local /work /net ( tmp_dir = {outdir}/ )')
+        tmp_dir = str( Path("{outdir}/").resolve() )
     else:
-        print('(cmorize_target_var_files) NOT using /local /work /net (tmp_dir = outdir/tmp/ )')
-        tmp_dir = f"{outdir}/tmp/"
+        print(f'(create_tmp_dir) NOT using /local /work /net (tmp_dir = {outdir}/tmp/ )')
+        tmp_dir = str( Path(f"{outdir}/tmp/").resolve() )
```

Comment:
we agreed to revisit this tmpdir and if it's needed at all. It seems odd to combine the string and path type conversion in one line but I've had to do it too.

Reply:
see #240 ! but perhaps we should discuss the best way to use/handle `pathlib.Path` in our codebase... lest we get too deep in the hole!